util/CMakeLists.txt

```diff
@@ -36,16 +33,18 @@ if(BUILD_UTILITIES)
     make_utility(srconv      srconv_main.c)
 
     if (USE_VCPKG)
-        find_package(SampleRate REQUIRED)
-        set(LIBSAMPLERATE_LIBRARY SampleRate::samplerate)
-    else ()
-        find_library(LIBSAMPLERATE_LIBRARY NAMES samplerate libsamplerate-0)
+        # vcpkg includes a config find module
+        find_package(SampleRate CONFIG)
+        # set a found variable because vcpkg doesn't
```

Comment:
Again these copy+pasted comments can be omitted. They're not adding much

Reply:
Ok, removed!